app/pulp/app/serializers/__init__.py

```diff
@@ -6,8 +6,9 @@
 from pulp.app.serializers.fields import (ContentRelatedField, RepositoryNestedIdentityField,  # NOQA
     RepositoryRelatedField)  # NOQA
 from pulp.app.serializers.generic import (ConfigKeyValueRelatedField,  # NOQA
-    NotesKeyValueRelatedField)  # NOQA
+    NotesKeyValueRelatedField, ScratchpadKeyValueRelatedField)  # NOQA
```

Comment:
since you added ScratchpadKeyValueRelatedField would you mind updating repository serializer?